modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2555,13 +2554,23 @@ private void _executeObjectActions(
 	}
 
 	private void _fillDefaultValue(
-		long objectDefinitionId, Map<String, Serializable> values) {
+		String defaultLanguageId, long objectDefinitionId,
+		Map<String, Serializable> values) {
 
 		for (ObjectField objectField :
 				_objectFieldPersistence.findByObjectDefinitionId(
 					objectDefinitionId)) {
 
-			if (values.get(objectField.getName()) != null) {
+			boolean localized = objectField.isLocalized();
```

Comment:
@pedro-oliveira446 do we really need to create this variable?

Reply:
Done!